core/services/offchainreporting/oracle.go

```diff
@@ -177,10 +189,16 @@ func (d jobSpawnerDelegate) ServicesForSpec(spec job.Spec) ([]job.Service, error
 		}
 
 	} else {
-		ocrkey, exists := d.keyStore.DecryptedOCRKey(concreteSpec.EncryptedOCRKeyBundleID)
+		ocrkey, exists := d.keyStore.DecryptedOCRKey(*concreteSpec.EncryptedOCRKeyBundleID)
```

Comment:
This makes me a little bit sad. Do we need a nil check?

Reply:
the validator does assert that for a nonboostrap node this has to be set, but it couldn't hurt to add another check